pubspec.yaml

```diff
@@ -23,6 +23,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  intl: ^0.19.0
+  mask_text_input_formatter: ^2.9.0
```

Comment:
i think in the future we should consider other ways to do this text masking - it would be great to not rely on this package long-term. 
for now it is fine